doc/gov/chapters/chapter01.md

```diff
@@ -11,18 +11,18 @@
 
-In order to establish governing guidelines and procedures by which the CNTT technical community may operate, the development of this document was undertaken by a CNTT working group and developed under the guidance of the CNTT Governance Steering Committee. 
+In order to establish governing guidelines and procedures by which the CNTT technical community may operate, the development of this document was undertaken by a CNTT workstream group and developed under the guidance of the CNTT Governance Steering Committee. 
 
 <a name="1.2"></a>
 ## 1.2 Document Purpose
 
-The goal of this document is to assist CNTT to continue to scale, garner momentum, and plan for the future of CNTT--as outlined in its 3rd Technical Workshop in Antwerp, Belgium, in September 2019. Therefore, the purpose of this document is to define and detail basic CNTT community operations and important structures. It provides a reference of the organization and related work streams, processes, and procedures around which the CNTT Community will be managed; including operational scope, structure, and responsibilities of all groups and individuals associated with CNTT.  
+The goal of this document is to assist CNTT to continue to scale, garner momentum, and plan for the future of CNTT--as outlined in its 3rd Technical Workshop in Antwerp, Belgium, in September 2019. Therefore, the purpose of this document is to define and detail basic CNTT community operations and important structures. It provides a reference of the organization and related work streams, processes, and procedures around which the CNTT Community is managed; including operational scope, structure, and responsibilities of all groups and individuals associated with CNTT.  
 
 This document is intended for use by CNTT Task Force's new and existing contributors, participants, associated organizations, and, as an open-source community following the GSMA and LFN Anti-Trust Policies and Creative Commons Licensing, the public at-large.
 
-This document is not intended to be used to define any the technical specifications, definitions, requirements, testing, certifications, or lab platforms developed by CNTT. 
+This document is not intended to be used to define any technical specifications, definitions, requirements, testing, certifications, or lab platforms developed by CNTT. 
```

Comment:
since it says "not intended to be used to define" (stress "to define") is it appropriate then to say "developed by CNTT"?

May I suggest a simplification: "This document does not define any technical specifications, definitions, requirements, architectures, testing, certifications, or lab platforms."